app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/Database/Device/DeviceDatabase.java

```diff
@@ -0,0 +1,66 @@
+package edu.drexel.lapcounter.lapcounter.backend.Database.Device;
+
+import android.arch.persistence.db.SupportSQLiteDatabase;
+import android.arch.persistence.room.Database;
+import android.arch.persistence.room.Room;
+import android.arch.persistence.room.RoomDatabase;
+import android.content.Context;
+import android.os.AsyncTask;
+import android.support.annotation.NonNull;
+
+import edu.drexel.lapcounter.lapcounter.backend.Database.Workout.WorkoutDao;
+import edu.drexel.lapcounter.lapcounter.backend.Database.Workout.Workouts;
+
+
+@Database(entities = {Device.class}, version = 1)
```

Comment:
Why do we need two `RoomDatabase`s? Couldn't we just have one `LapCounterDatabase` that holds the entities `Device` and `Workout`?

Reply:
```suggestion
@Database(entities = {Device.class, Workouts.class}, version = 1)
```

Reply:
I see no problem is doing this and just having one if it works.